tests/lib/rules/no-tabs.js

```diff
@@ -24,10 +24,13 @@ ruleTester.run("no-tabs", rule, {
         "function test(){\n" +
         "  //   sdfdsf \n" +
         "}",
+        "var str = 'test\tstring'",
+        "var re = /test\tstring/"
```

Comment:
To leave how it's fixed to developers.
Removing might be an unexpected change and the developer may not be aware of the change because `eslint --fix` does not report fixed problems.
On the other hand, if ESLint remained the problem, `eslint --fix` reports the problem. Then developers can fix it by the proper way.